rocky/assets/css/components/report.scss

```diff
@@ -95,3 +95,15 @@
     }
   }
 }
+
+span {
+  &.label {
+    display:inline-flex;
+    max-width:300px;
+    border-radius:4px;
+    padding:4px 8px;
+    font-size: var(--tile-font-size);
+    font-weight:700;
```

Comment:
If we set a max-width, we also need an overflow, and possibly a tooltip/title argument for those that are too long.

Reply:
Yes, the inline-flex will fix that